Projects/Server/Items/Item.cs

```diff
@@ -4123,6 +4124,20 @@ public virtual void OnAosSingleClick(Mobile from)
         }
     }
 
+    public static void AppendWithSpace(StringBuilder builder, string text)
+    {
+        if (!string.IsNullOrEmpty(text))
+        {
+            if (builder.Length > 0) builder.Append(" ");
+            builder.Append(text);
+        }
```

Comment:
```suggestion
        if (string.IsNullOrEmpty(text))
        {
            return;
        }
        if (builder.Length > 0) builder.Append(" ");
        builder.Append(text);
```

as a best practice we try to use minimal nesting in newly introduced code for clarity